### DISCUSSION TITLE
Private container registry support

### DISCUSSION THREAD

**Discussion by vadakr** [vmware-tanzu/velero#4433] [Community support Q&A]
Are there any install options that control the registry to pull (velero) container images from? We'd like to use a private registry instead of Docker Hub.

**Comment by Satsank** [comment]
Answered this earlier offline to @vadakr... Velero Helm Chart has support for alternate repository...

https://github.com/vmware-tanzu/helm-charts/blob/daa10dd/charts/velero/values.yaml#L15-L16
